src/util/fees.cpp

```diff
@@ -26,16 +29,32 @@ std::string StringForFeeReason(FeeReason reason) {
     return reason_string->second;
 }
 
-bool FeeModeFromString(const std::string& mode_string, FeeEstimateMode& fee_estimate_mode) {
-    static const std::map<std::string, FeeEstimateMode> fee_modes = {
+const std::vector<std::pair<std::string, FeeEstimateMode>>& FeeModeMap() {
+    static const std::vector<std::pair<std::string, FeeEstimateMode>> fee_modes = {
         {"UNSET", FeeEstimateMode::UNSET},
         {"ECONOMICAL", FeeEstimateMode::ECONOMICAL},
         {"CONSERVATIVE", FeeEstimateMode::CONSERVATIVE},
     };
-    auto mode = fee_modes.find(mode_string);
+    return fee_modes;
+}
 
-    if (mode == fee_modes.end()) return false;
+std::string FeeModes(const std::string& delimiter) {
```

Comment:
457396a4091bb3c27b1e982fb7e08442120b15df

This is a utility function for the RPC help, I'd move to `rpcwallet.cpp` and make it static there.

BTW, according to dev notes

> Braces on new lines for classes, functions, methods.

Reply:
Fixed style.

Reply:
Forgot to respond to the first point; I could move it, but it would mean having to make `fee_modes` available to the public, which doesn't seem worth it.